schemas/context/segmentmembershippropensitypayload.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentMembershipPropensityPayload",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Membership Propensity Payload",
+  "type": "object",
```

Comment:
I had tried to make this schema simply by not having it be an object type, but object type is enforced by XDM guidelines. The alternative would be for it to be a number and just define the min and max.